test/cases/api.sh

```diff
@@ -895,6 +895,15 @@ function verifyInAWSS3() {
   local S3_URL
   S3_URL=$(echo "$UPLOAD_OPTIONS" | jq -r '.url')
 
+  # Tag the resource as a test file
+  local S3_FILENAME
+  S3_FILENAME=$(echo "${S3_URL}" | grep -oP '(?<=/)[^/]+(?=\?)')
+
+  $AWS_CMD s3api put-object-tagging \
+	  --bucket "${AWS_BUCKET}" \
+	  --key "${S3_FILENAME}" \
+	  --tagging '{"TagSet": [{ "Key": "gitlab-ci-test", "Value": "true" }]}'	  
+
```

Comment:
What about aws.sh and api_v2.sh scripts ? Don't they need the same tagging ?

Reply:
`aws.sh` was updates in a previous PR https://github.com/osbuild/osbuild-composer/blob/main/test/cases/aws.sh#L228 
Idk about `api_v2.sh` it's not even being ran in the CI...

Reply:
`api_v2.sh` was removed in ea2812f48f29d1e7bed9b2ef9b24bb8094659d98